extensions/amp-subscriptions/0.1/local-subscription-platform-remote.js

```diff
@@ -60,6 +60,9 @@ export class LocalSubscriptionRemotePlatform extends LocalSubscriptionBasePlatfo
     return this.urlBuilder_
       .buildUrl(this.authorizationUrl_, /* useAuthData */ false)
       .then(fetchUrl => {
+        // WARNING: If this key is really long, you might run into issues by hitting
+        // the maximum URL length in some browsers when sending the GET fetch URL.
+        // If you run into this issue, it would be best to use POST.
```

Comment:
I'd remove the "If you run into this issue, it would be best to use POST." part.  A) users can't change this, B) if we do change this it will complicate things with CORS so we don't want to imply it's an easy change when it isn't.   Leave the first 2 lines.

Reply:
Done.